src/src/CustomBuild/ESPEasyDefaults.h

```diff
@@ -320,12 +320,18 @@
 #endif
 
 #ifndef GITHUB_RELEASES_LINK_PREFIX
-# define GITHUB_RELEASES_LINK_PREFIX "<a href='https://github.com/letscontrolit/ESPEasy/releases' title='Click to show latest release on Github' target='_blank' style='font-size: 15px; text-decoration: none'>"
```

Comment:
Why not a class here?

Reply:
because it takes precious memory bytes, and as this <A> is included only into the <div class='build'>, it can be styled easely from the CSS, if needed...